example/src/main/java/com/stripe/example/activity/StripeIntentActivity.kt

```diff
@@ -171,34 +174,18 @@ abstract class StripeIntentActivity : AppCompatActivity() {
         keyboardController.hide()
 
         viewModel.status.value += "\n\nPayment authentication completed, getting result"
-        val isPaymentResult =
-            stripe.onPaymentResult(
-                requestCode,
-                data,
-                object : ApiResultCallback<PaymentIntentResult> {
-                    override fun onSuccess(result: PaymentIntentResult) {
-                        viewModel.paymentIntentResultLiveData.value = Result.success(result)
-                    }
-
-                    override fun onError(e: Exception) {
-                        viewModel.paymentIntentResultLiveData.value = Result.failure(e)
-                    }
+        if (stripe.isPaymentResult(requestCode, data)) {
```

Comment:
can you add a null check for data so we don't need to use `!!`?

Reply:
`stripe.isPaymentResult` already checks nullability within, I think it's worth bring to the client's attention it does that. Added a comment here, wdty?

Reply:
This is fine, just thinking that there might be some future scenario where `isPaymentResult` stops checking for `data`'s nullity and this crashes. Or maybe `getPaymentIntentResult` should just take a `Intent?`?